rbac/api/cross_access/view.py

```diff
@@ -365,13 +365,9 @@ def check_update_permission(self, request, update_obj):
                 "Please use PATCH to update the status of the request.",
             )
 
-        # Do not allow updating the target_account.
-        if settings.AUTHENTICATE_WITH_ORG_ID:
-            if request.data.get("target_org") and str(request.data.get("target_org")) != update_obj.target_org:
-                self.throw_validation_error("cross-account-update", "Target account must stay the same.")
-        else:
-            if (
-                request.data.get("target_account")
-                and str(request.data.get("target_account")) != update_obj.target_account
-            ):
-                self.throw_validation_error("cross-account-update", "Target account must stay the same.")
+        # Do not allow updating the target_account or target_account.
```

Comment:
Small nit, we can't update the target_account or the target_account, but what about the target_account? Or, y'know, target_org. 😄

Reply:
I want to be REALLY sure that target_account can't be updated lol.
I just pushed the correction.